src/Api/include/Api/Orbit.h

```diff
@@ -316,14 +316,14 @@ extern ORBIT_EXPORT void* orbit_api_get_function_table_address_v0();
   orbit_api_v0 g_orbit_api_v0; \
   void* orbit_api_get_function_table_address_v0() { return &g_orbit_api_v0; }
 
-inline bool orbit_api_active() {
-  bool active = g_orbit_api_v0.active;
+inline bool orbit_api_enabled() {
+  bool enabled = g_orbit_api_v0.initialized && g_orbit_api_v0.enabled;
```

Comment:
Personal preference: In a way I preferred "active" as the name of the local variable to distinguish between the result of the `&&` and `g_orbit_api_v0.enabled` itself.

Reply:
Good, point, done.